DuckDuckGo/TabViewController.swift

```diff
@@ -173,8 +174,13 @@ class TabViewController: UIViewController {
         instrumentation.willPrepareWebView()
         webView = WKWebView(frame: view.bounds, configuration: configuration)
         webView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
-        webView.allowsLinkPreview = false
-        attachLongPressHandler(webView: webView)
+        
+        if #available(iOS 13, *) {
+            attachLongPressHandler(webView: webView)
```

Comment:
Shouldn't this be attached for iOS 12 instead?

Reply:
Oops - was rushing to push it before heading to dinner 🤦‍♂